interface/src/components/swap/quote-info.tsx

```diff
@@ -201,6 +213,21 @@ export const QuoteInfo = (props: Props) => {
           </Bubble>
         </Row>
       )}
+      {selectedQuoteOption?.braveFee && realBraveFee && (
+        <Row rowWidth='full' marginBottom={16} horizontalPadding={16}>
+          <Text textSize='14px'>{getLocale('braveSwapBraveFee')}</Text>
+          <Text textSize='14px'>
+            <BraveFeeContainer>
+              {realBraveFee.isZero() && <BraveFeeFree>{getLocale('braveSwapFree')}</BraveFeeFree>}
+              {realBraveFee.isZero() ? (
+                <BraveFeeDiscounted>{`${selectedQuoteOption.braveFee.fee}%`}</BraveFeeDiscounted>
+              ) : (
+                `${realBraveFee.format()}%`
+              )}
+            </BraveFeeContainer>
```

Comment:
Can we reuse our shared styles?

```suggestion
          <BraveFeeContainer>
            {realBraveFee.isZero() &&
              <Text textSize='14px' textColor='success' isBold={true}>{getLocale('braveSwapFree')}</Text>
            }
            {realBraveFee.isZero() ? (
              <BraveFeeDiscounted textSize='14px' textColor='text03'>
                {`${selectedQuoteOption.braveFee.fee}%`}
              </BraveFeeDiscounted>
            ) : (
              <Text textSize='14px' textColor='text03' isBold={true}>
              {`${realBraveFee.format()}%`}
              </Text>
            )}
          </BraveFeeContainer>
```